app/src/main/java/org/sopt/and/navigation/BottomNavItem.kt

```diff
@@ -0,0 +1,16 @@
+package org.sopt.and.navigation
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.material.icons.filled.Person
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.ui.graphics.vector.ImageVector
+import org.sopt.and.R
+
+sealed class BottomNavItem(
+    val title: Int, val icon: ImageVector, val route: String
```

Comment:
Android Resource id를 필요로 하는 필드는 `@StringRes`, `@DrawableRes`등으로 명시해주면, resource id가 필요한 필드임을 확실하게 명시해줄 수 있습니다 !
```suggestion
    @StringRes val title: Int,
    val icon: ImageVector,
    val route: String
```